### DISCUSSION TITLE
Anyway to transfer playback statstics and playlists from Foobar2000?

### DISCUSSION THREAD

**Discussion by funkyblue** [strawberrymusicplayer/strawberry#1294] [Q&A]
Hello,

So I've used Foobar2000 for over 14 years. 14 years of perfect playlist management and playback statistics and also uploads to Last.FM. 

Is there a way I can import this data into Strawberry Music Player? The answer is always no when I try and switch to other products. But I thought I'd check.

Kind Regards

**Comment by jonaski** [comment]
https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
